posts/238-new-make-shuffle-mode.md

```diff
@@ -8,6 +8,20 @@ date: February 23, 2022
 I implemented new `--shuffle` option for `GNU make` to simulate
 non-deterministic build order in parallel makefiles.
 
+**UPDATE**: `--shuffle` mode was released as `GNU make 4.4` and can be
+used as:
+
+- `make --shuffle ...` from the command line or via, for interactive
+  use.
+- `GNUMAKEFLAGS=--shuffle` via environment variables for wider scale
+  builds.
+
+[Options Summary](https://www.gnu.org/software/make/manual/html_node/Options-Summary.html)
+has more details.
+
+It already found bugs in [30+ packages](/posts/249-an-update-on-make-shuffle.html)
+like `gcc`, `vim`, `ghc`, `subversion`, strace`, ispell` and others.
```

Comment:
"strace" and "ispell" have unbalanced backticks:
```diff
--- strace`, ispell`
+++ `strace`, `ispell`
```

Reply:
Good catch! Updated as well.